### PR TITLE
Bump rubocop to v0.53.0

### PR DESCRIPTION
**Pull request by timrogers**

### CODE REVIEW

**Comment by timrogers** [comment]
@hmac Could you give this a review please? We'll need to release a new version to update projects using this to the new Rubocop.

**Comment by hmac** [comment]
@timrogers sorry! Meant to get to this yesterday but dropped it

File: validate_config.rb [hunk 1]
```diff
@@ -4,13 +4,32 @@
...
 
-# Exit with non-zero code so CI will mark as failed
-exit(1) unless $stderr.string.empty?
+  yield
+
+  $stdout = default_stdout
+  $stderr = default_stderr
+
+  [redirected_stdout, redirected_stderr]
+end
```
Comment (hmac) [inline validate_config.rb]: You can probably shorten this by assuming that `$stdout == STDOUT`
```ruby
[StringIO.new, StringIO.new].tap do |out, err|
	$stdout, $stderr = out, err

	yield

	$stdout, $stderr = STDOUT, STDERR
end
```

Comment (timrogers) [reply validate_config.rb]: Ah, I didn't know about those constants. Will do.

**Pull request merged**